documentation/suggested-project-structure.md

```diff
@@ -0,0 +1,97 @@
+### Suggested Project Structure
+
+ * `server.js` is the main entry point.
+ * Think of the `resources` folder as your `routes`.
+ * Think of the `handlers` folder as your `controllers`.
+
+```
+├── handlers
+│   ├── articleHandler.js
+│   ├── commentHandler.js
+│   ├── peopleHandler.js
+│   ├── photoHandler.js
+│   └── tagHandler.js
+├── resources
+│   ├── articles.js
+│   ├── comments.js
+│   ├── people.js
+│   ├── photos.js
+│   └── tags.js
+└── server.js          
+```
+
+#### Example server.js
+
+This is the main entry point for your API. The goal is to configure jsonapi-server and load all the resources.
+
+```javascript
+var jsonApi = require("jsonapi-server");
+var fs = require("fs");
+var path = require("path");
+
+jsonApi.setConfig({
+  // Put your config here!
```

Comment:
do you have examples of what the config might look like? e.g what params are required? You could write:
```
jsonApi.setConfig({
  // host: 'http://foobar.com,
  // port: '5000',
  // foo: 'bar'
})
```

By providing the required config with example values, but commented out, people can more easily pick this up and run with it.